src/topp/DatabaseSuitability.cpp

```diff
@@ -95,7 +94,7 @@ To generate the de novo "database":
       - @ref TOPP_IDFileConverter generates the de novo fasta file.
 
 For re-ranking all cases where a peptide hit only found in the de novo "database" scores above a peptide hit found in the actual database are checked. In all these cases the cross-correlation scores of those peptide hits are compared. If they are similar enough, the database hit will be re-ranked to be on top of the de novo hit. You can control how much of cases with similar scores will be re-ranked by using the @p novor_fract option.@n
-For this to work it is important that FDR filtering is done in this tool and not beforehand by @ref TOPP_FalseDiscoveryRate. You can provide the wanted FDR using the corresponding flag.
+For this to work it is important that no FDR was performed. This tool does this itself and will crash if a q-value is found. You can still control the FDR that you want to establish using the corresponding flag.
```

Comment:
Mention that PepIndexer is required (its only mentioned in the parameter help so far).